articles/libraries/lock/v11/migration-guide.md

```diff
@@ -47,6 +47,12 @@ The documents below describe all the changes that you should be aware of when mi
 If you have any questions or concerns, you can discuss them in the [Auth0 Community](https://community.auth0.com/), submit them using the [Support Center](${env.DOMAIN_URL_SUPPORT}), or directly through your account representative, if applicable. 
 :::
 
+## Disabling Legacy Lock API
+
+After you update to Lock 11 and Auth0.js v9, it is advised that you turn off the **Legacy Lock API** toggle in the Dashboard. This will make your Auth0 tenant behave as if the API is no longer available, so you can make sure you completed the required changes successfully. You can find the setting in the [Advanced section](${manage_url}/#/tenant/advanced) of Tenant Settings.
```

Comment:
Looks good, and I would also add something like 

"Starting on April 1, this option will be forcibly disabled, so it is recommended you opt-in before that time to verify your configuration will work correctly."

Or something like that to let people know what we will be doing.